app/cafe.py

```diff
@@ -0,0 +1,30 @@
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not ("vaccine" in visitor):
+            raise NotVaccinatedError(
+                "NotVaccinatedError, Person should be vaccinated!"
+            )
+        if datetime.date.today() > visitor.get(
+                "vaccine"
+        ).get(
+            "expiration_date"
+        ):

```

Comment:
Here you have two errors:
visitor.get() can return none
visitor.get("vaccine").get() can return None and than you compare